apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -314,6 +319,13 @@ def _get_nrf_device_args(options):
 ]
 
 
+def get_zephyr_base(options: dict):
+    """Returns Zephyr base path"""
+    zephyr_base = options.get("zephyr_base", ZEPHYR_BASE)
+    assert zephyr_base, "'zephyr_base' not passed and not found by default!"
```

Comment:
Maybe s/not passed/option not passed/ here to match what's in the Arduino's code?